<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMinimisation</class>
 <widget class="QDialog" name="FormMinimisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Экстракция параметров</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="6">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QTabWidget" name="tabGraphics">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Ib-Ub</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Ic-Uc</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnSavePlot">
       <property name="text">
        <string>Сохранить график</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTextBrowser" name="logger">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff325d;&quot;&gt;Начальные параметры:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;BF=1, AR=10&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff842d;&quot;&gt;Итерация #1&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Шаг=100&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff1210;&quot;&gt;Ошибка: &lt;/span&gt;Нехватает данных&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c0a22a;&quot;&gt;Предупреждение:&lt;/span&gt;&lt;span style=&quot; color:#ffeb39;&quot;&gt; &lt;/span&gt;нехватает параметра&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff325d;&quot;&gt;Оптимальные параметры:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;BF=0.5, AR=1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Статус:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Работает</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Итерация:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelIterations">
       <property name="text">
        <string>1/30</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ошибка:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelError">
       <property name="text">
        <string>0.011</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QTableView" name="tblParameters"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnSaveParameters">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressMinimisation">
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOperation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Запуск</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCopyToClipboard">
   <property name="text">
    <string>Скопировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <zorder>btnSaveParameters</zorder>
  <zorder>tblParameters</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>FormMinimisation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
